x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/graph_investigation/graph_investigation.tsx

```diff
@@ -167,6 +169,21 @@ interface GraphInvestigationProps {
      */
     timeRange: TimeRange;
   };
+
+  /**
+   * Whether to show investigate in timeline action button. Defaults value is false.
+   */
+  showInvestigateInTimeline?: boolean;
```

Comment:
```suggestion
  showInvestigateInTimeline: boolean;
```

Since we are defaulting it in the component to false we don't need `true 
|| false || undefined` as the type.

Reply:
no can do mister, it forces to use this property (fails on typescript check)